test/JDBC/expected/BABEL-USER-vu-verify.out

```diff
@@ -203,6 +203,39 @@ babel_user_vu_prepare_test3#!#babel_user_vu_prepare_sch
 ~~END~~
```

Comment:
Please add testcases 
1. which establishes a session with new mapping.
2. about cross db query. - Because cross-db query are supported using overriding the session with current_login(), so it would be good to verify that internal pg role membership is intact.

Reply:
Already have existing test cases for this but added more.